core/rff_weather_indeces.mcf

```diff
@@ -0,0 +1,15 @@
+Node: heavyPrecipitationIndex
```

Comment:
Can we please add these properties here: https://github.com/datacommonsorg/schema/blob/main/core/climate.mcf#L340 ?